src/App.js

```diff
@@ -1,77 +1,71 @@
-import React from 'react';
+import {useState, useEffect} from 'react';
+import ReactGA from 'react-ga';
 import { div, h } from 'react-hyperscript-helpers';
 import Modal from 'react-modal';
 import './App.css';
 import DuosFooter from './components/DuosFooter';
 import DuosHeader from './components/DuosHeader';
+import {useHistory} from 'react-router-dom';
 
 import { SpinnerComponent as Spinner } from './components/SpinnerComponent';
 import { Storage } from './libs/storage';
 import Routes from './Routes';
 
+function App() {
+  const [loading] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  let history = useHistory();
 
-class App extends React.Component {
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      loading: false,
-      isLoggedIn: false
-    };
-  }
-
-  showSpinner = () => {
-    this.setState({
-      loading: true
+  const initializeReactGA = () => {
+    ReactGA.initialize('UA-173273990-1', {
```

Comment:
We should pull this from configs similarly to how we grab other keys like `Config.getGoogleClientId()` The complication here is that we have to wait for the app to load completely before doing so, similarly to how we wait for the config object to load before enabling the sign-in button.